CLDR-15370 Enhance Info Hub with new features and guidelines (#5611)
unicode-org/cldr: docs/site/translation.md

There are 4 new items that are used in constructing locale names.
When text containing parentheses is embedded in _other_ parentheses, 
these are bracket characters used to fix the embedded parentheses.

For example, this is used in locale names such as the locale name for `en_MM`.
The name for the language (such as “anglais”) is composed with the name of the region 
(such as “Myanmar (Birmanie)”) with the `localePattern` “{0} ({1})”, 
but the parentheses in the language or region are replaced using these Nested Bracket Replacements,
to get “anglais (Myanmar \[Birmanie])”.

**TBD** Add examples in the Survey Tool so people can see the difference it makes.

##### Guidelines

These characters used to be hard-coded for certain locales 
(typically ones using full-width parentheses and brackets), 
but now they can be customized if needed for your locale.

#### Territories

Added long display names for the territories 
"St. Helena, Ascension & Tristan da Cunha" (`SH`),
"French Southern and Antarctic Lands" (`TF`),
and "Heard Island & McDonald Islands" (`HM`).
The previous names were moved to `alt="short"`.

##### Guidelines

Revisit these items to make sure that the values are correct.

> #### Additional Locale Display Names—Keys

Author: @srl295 I just redid this section, please take a look.

Reviewer: looks good!

your locale always uses a form like "March 3rd, 2026".
In that case, for the code `yMMMMd` you would change its pattern to have `ddd` in it to force the use of ordinals, something like: "MMMM ddd, y"
* If your locale _sometimes_ uses ordinals with **non-numeric months**,
then generally when a skeleton has `ddd` in it, the pattern should also have it;
when a skeleton has `d` in it, the pattern should also have it;
    * However, review the results as there may be some patterns where ordinals are disallowed or required.

#### Numeric separators (date & time)

(2 items)

There are two new items used in pure-numeric dates and times, such as 03/04/2026 or 13:45:30.
For these, the values would be "/" and ":". 

##### Guidelines

Make sure these match the typical characters used in pure-numeric formats of dates and times in your locale. If more than one is commonly used in your locale, please use the separator that matches the current date and time formats in the CLDR.

#### Dual Standard/Daylight format

(1 item)

For the generic offset time zone (one without a specific name like “Pacific Time”), there have been only patterns like “UTC+8”.

> This new pattern gives people more information, by showing values like “UTC+8/+7” for time zones that have daylight time.

Author: ```suggestion
One of the formats for timezones is to list the offsets from UTC.
That works well for places that don't have a distinct daylight time (aka summer time).
- Nigeria Time → UTC+1
- South Africa Time → UTC+2

That doesn't work well for places that alternate between standard and daylight times, such as CET.
A majority of the year they are not on a standard time, but rather one hour ahead.
A new localizable pattern allows for a more informative representation, such as:
- Central European Time — UTC+1/+2

This is done with a pattern such as “{0}/{1}" that combines the two offsets, 
and is then substituted into the `gmtFormat`, which has localized versions of “UTC{0}" (or “GMT{0}").
```

> <img width="521" height="172" alt="Screenshot 2026-04-21 at 16 30 05" src="https://github.com/user-attachments/assets/3bd4d252-4f18-4e08-9c43-0341f7557c27" />

Reviewer: ```suggestion
```

The preview didn't deploy because of this line. If it's supposed to be in this file, it should be further down.

The Survey Tool has opened early [DDL locales] data submission. See the [DDL: Help Center] for more information on submitting data for DDL locales.

Languages which have a value in the *Target Level* column of the [Locale Coverage chart] are considered *TC locales* and will not open for submission until the start of the next regular submission for CLDR 49 in April or May of 2026.

- **Disconnect error**. If you see a persistent Loading error with a disconnect message or other odd behavior, please [empty your cache](translation/getting-started/empty-cache).
- Survey Tool email notification may be going to your spam folder. Check your spam folder regularly.
- "**Same as code**" errors - when translating codes for items such as languages, regions, scripts, and keys, it is normally an error to select the code itself as the translated name.
If the error appears under Typography, you can ignore it. <!-- [[CLDR-13552](https://unicode-org.atlassian.net/browse/CLDR-13552)\] -->

## New languages

We are reviewing new locale requests for inclusion in CLDR 49. See [how to add a new locales](https://cldr.unicode.org/development/adding-locales).

## New Areas

### Locale display names

> #### Nested Bracket Replacement

Reviewer: see https://cldr.unicode.org/downloads/cldr-48#spec-changes-1 (which has broken links)

Author: good point. We introduced it then, but this is the first time vetters have seen it.

(4 items)

There are 4 new items that are used in constructing locale names.
When text containing parentheses is embedded in _other_ parentheses, 
these are bracket characters used to fix the embedded parentheses.

For example, this is used in locale names such as the locale name for `en_MM`.
The name for the language (such as “anglais”) is composed with the name of the region 
(such as “Myanmar (Birmanie)”) with the `localePattern` “{0} ({1})”, 
but the parentheses in the language or region are replaced using these Nested Bracket Replacements,
to get “anglais (Myanmar \[Birmanie])”.

**TBD** Add examples in the Survey Tool so people can see the difference it makes.

##### Guidelines

These characters used to be hard-coded for certain locales 
(typically ones using full-width parentheses and brackets), 
but now they can be customized if needed for your locale.

> #### Territories

Reviewer: for more, see https://cldr.unicode.org/translation/displaynames/countryregion-territory-names

Author: I deleted a previous comment, that was really for another section.

Author: That page really doesn't talk about the use of brackets, so I don't think we want to point at it.

Reviewer: > That page really doesn't talk about the use of brackets, so I don't think we want to point at it.

This is the Territories line, not the brackets?

For example, there is a key `lb` for CJK Line Break behavior that has various options. 
The Code value shows just the key, or the key-option or key-option-core

| Code | English | Native |
| :---- | :---- | :---- |
| lb | CJK Line Break | style de saut de ligne |
| lb-loose | Loose Line Break Style | style de saut de ligne permissif |
| lb-loose-core | Loose | permissif |
| lb-normal | Normal Line Break Style | style de saut de ligne normal |
| lb-normal-core | Normal | normal |

The key-option-core value is the one that would be used in a menu or pulldown list,
where the header is the key, and the sub-elements are key-option-core values, such as:

* CJK Line Break
    * Loose
    * Normal
 
The key-option-core value can also be used in the full name of the locale code
using a pattern to combine it with the key, such as “Chinese (Singapore, CJK Line Break: Loose)”. 

The key-option value combines the name of the instruction with the name of the option. 
It is less versatile, since it looks overly verbose in menus. 
CLDR is focusing on gathering the names of the key-option-core options; 

> the others provided mostly for reference.

Reviewer: This section, for example, is parallel to https://cldr.unicode.org/translation/displaynames/locale-option-names-key but doesn't link to it, and uses different terminology and examples.

##### Guidelines

Where there are key-option values, you can use that to guide your name of the key-option-core value
— basically removing the name of the key.
Otherwise, go by the English name for the key-option values.

### Dates and times

#### Ordinal days in dates

🆕 2026-04-18

In some locales, ordinal numbers (such as 1st, 2nd, …) can be used in dates. 
For example, ordinal: "March 3rd, 2026" compared to cardinal: "March 3, 2026". 

> There are now two new types of data items to support this.

Reviewer: https://cldr.unicode.org/translation/date-time/date-time-patterns#week-of-patterns still says "in the future we may support ordinals…"

Author: Just removed that line (I didn't change it to talk about this, because that section is about week of months.)

   * The exact items will depend on the locale: many locales have a "constant" pattern, such as German.
In that case only the `other` code will appear.
   * A few locales didn't have ordinal categories, just plural categories. That is being corrected in this release.
* Date & Time | Gregorian | Formats-Flexible-Date_Formats | yMMMddd — English: `MMM ddd, y`
   * The symbol `ddd` in this pattern will be replaced by an ordinal, resulting in something like "Sep **3rd**, 1999"

##### Guidelines

For DayOfMonth-abbreviated-Formatting:

* These ordinal forms are _specific to dates_; they are _not_ general-purpose.
* They should have the appropriate grammatical form for a nominative date.
* They might not be the same as general-purpose ordinals.
    * For example, suppose that your locale uses a "er" suffix just on `one` in dates. 
    * In that case, the `one` form would be {0}er, but all other forms would have just {0}.
* If your locale _never_ uses ordinals in dates, then:
    * Set all the dayOfMonth patterns (`one`, `other`, …) to a constant “{0}” with no other text.

For Formats-Flexible-Date_Formats

* The `ddd` is ignored in any pattern with a _numeric_ month (M, MM).

> It will only appear and be used with _non-numeric_ months (MMM, MMMM).

Reviewer 2: ```suggestion
It will only appear and be used with _non-numeric_ months (MMM, MMMM). For example, Dec or December. [See Date/Time Symbols](/date-time/date-time-symbols) for more information about symbol length.
```

* If your locale _always_ uses ordinals with **non-numeric months**, then make sure the patterns where they are used _always_ have `ddd` in them (instead of `d` or `dd`).
    * For example, suppose that a form like "March 3, 2026" is not acceptable;
your locale always uses a form like "March 3rd, 2026".
In that case, for the code `yMMMMd` you would change its pattern to have `ddd` in it to force the use of ordinals, something like: "MMMM ddd, y"
* If your locale _sometimes_ uses ordinals with **non-numeric months**,

> then generally when a skeleton has `ddd` in it, the pattern should also;

Reviewer 3: ```suggestion
then generally when a skeleton has `ddd` in it, the pattern should also have it;
```

> when when a skeleton has `d` in it, the pattern should also;

Reviewer 3: ```suggestion
when a skeleton has `d` in it, the pattern should also have it;
```

>     * However, review the results because there may be some patterns where ordinals are disallowd or required.

Reviewer 3: ```suggestion
    * However, review the results as there may be some patterns where ordinals are disallowed or required.
```

### Nested Bracket Replacement

🆕 2026-04-20

There are 4 new items that are used in constructing locale names.
When text containing parentheses is embedded in _other_ parentheses, 
these are bracket characters used to fix the embedded parentheses.

For example, this is used in locale names such as the locale name for `en_MM`.
The name for the language (such as “anglais”) is composed with the name of the region 
(such as “Myanmar (Birmanie)”) with the `localePattern` “{0} ({1})”, 
but the parentheses in the language or region are replaced using these Nested Bracket Replacements,
to get “anglais (Myanmar \[Birmanie])”.

**TBD** Add examples in the Survey Tool so people can see the difference it makes.

##### Guidelines

These characters used to be hard-coded for certain locales 
(typically ones using full-width parentheses and brackets), 
but now they can be customized if needed for your locale.

### Numeric separators (date & time)

🆕 2026-04-20

There are two new items used in pure-numeric dates and times, such as 03/04/2026 or 13:45:30.
For these, the values would be "/" and ":". 

##### Guidelines

> Make sure these match the typical characters used in pure-numeric formats of dates and times in your locale.

Reviewer 2: ```suggestion
Make sure these match the typical characters used in pure-numeric formats of dates and times in your locale. If more than one is commonly used in your locale, please use the separator that matches the current date and time formats in the CLDR.
```

Reviewer 3: "the CLDR"?

> Make sure these match the typical characters used in pure-numeric formats of dates and times in your locale.

Reviewer 2: Are we going to lock these in future cycles similar to group separator etc?

Author: Yes, we should.

### Dual Standard/Daylight format

🆕 2026-04-20

For the generic offset time zone (one without a specific name like “Pacific Time”), there have been only patterns like “UTC+8”.

> This new pattern gives people more information, by showing values like “UTC+8/+7” for time zones that have daylight time.

Reviewer 2: Maybe we should give an example here about regions where some parts of a region observe daylight savings and others don't?

Author: Good idea: that one is for Pacific time: we can give another example for one that doesn't vary (Japan?)

Author: Or use Pacific Time, Arizona Time, and Denver Time.

Reviewer 3: Or maybe be less US centric and use country examples like India or Brazil as examples of countries that do not change time, and examples like Germany or France as examples of those who do, and examples like US and Australia as examples of mixed bags.

Author: That works too; I was thinking of an example where one country has UTC+X/+Y and another has just UTC+X. Maybe a European country and one in Africa?

Author: Here's a non-US example:

Central European Time — UTC+1/+2
Nigeria Time — UTC+1
South Africa Time — UTC+2

##### Guidelines

> Use a punctuation character in the pattern that expresses in your locale that a time zone has two alternate timezone offsets

Reviewer 3: ```suggestion
Use a punctuation character in the pattern for your locale that shows that a time zone has two alternate timezone offsets
```

(one in summer and one in winter).

### Additional available skeletons

Aside from the new skeletons with `ddd` used for Ordinal days in dates, 

> there are some new patterns that flesh out support for different combinations of long months (MMMM) plus days, and perhaps eras or days of the week, such as `MMMMEd`.

Reviewer 3: ```suggestion
there are some new patterns that flesh out support for different combinations of long months (MMMM) plus days, and eras or days of the week, such as `MMMMEd`.
```

Reviewer 3: Why the ambiguous "perhaps" in this sentence? I'm not a subject matter expert and am not entirely sure what we are trying to convey here, so please re-review this sentence for clarity.

##### Guidelines

> Typically the format will be aligned with the format with the format for abbreviated months (MMM).

Reviewer 3: ```suggestion
Typically the format will be aligned with the format for abbreviated months (MMM).
```

So look for the corresponding skeleton to see what the pattern is, then modify it to have MMMM.
This is not done automatically, because in some locales the best format may be a bit different.

### Additional interval skeletons

> Like the *Additional available skeletons*, there are a few new interval skeletons;

Reviewer 3: ```suggestion
Like the *Additional available skeletons*, there are a few new interval skeletons.
```

> so check to make sure they have patterns that are similar to related interval skeletons' patterns.

Reviewer 3: ```suggestion
Check to make sure they have patterns that are similar to related interval skeletons' patterns.
```

### Additional Locale Display Names—Keys

> Locale codes are used not only for languages and regional or script variants,

Reviewer 3: ```suggestion
Locale codes are not only used for languages and regional or script variants,
```

but can also carry instructions for how to handle various services.
For example, a locale code can also specify a preferred number system, 

> which can cause number to show as either Western digits (012…) or Arabic-Indic digits (٠١٢…).

Reviewer 3: ```suggestion
which can cause numbers to show as either Western digits (012…) or Arabic-Indic digits (٠١٢…).
```

These instructions have various options associated with them; the types can be 'full' or 'core'.
For example, there is a key `lb` for CJK Line Break behavior that has various options. 
The Code value shows just the key, or the key-option or key-option-core

| Code | English | Native |
| :---- | :---- | :---- |
| lb | CJK Line Break | style de saut de ligne |
| lb-loose | Loose Line Break Style | style de saut de ligne permissif |
| lb-loose-core | Loose | permissif |
| lb-normal | Normal Line Break Style | style de saut de ligne normal |
| lb-normal-core | Normal | normal |

The key-option-core value is the one that would be used in a menu or pulldown list,
where the header is the key, and the sub-elements are key-option-core values, such as:

* CJK Line Break
    * Loose
    * Normal
 
The key-option-core value can also be used in the full name of the locale code

> using a pattern to combined it with the key, such as “Chinese (Singapore, CJK Line Break: Loose)”. 

Reviewer 3: ```suggestion
using a pattern to combine it with the key, such as “Chinese (Singapore, CJK Line Break: Loose)”. 
```

The key-option value combines the name of the instruction with the name of the option. 
It is less versatile, since it looks overly verbose in menus. 
CLDR is focusing on gathering the names of the key-option-core options; 
the others provided mostly for reference.

##### Guidelines

Where there are key-option values, you can use that to guide your name of the key-option-core value
— basically removing the name of the key.
Otherwise, go by the English name for the key-option values.

### New emoji

**TBD** - Section will be updated when Unicode 18 emoji keywords are added to the Survey Tool for localization

## Survey Tool

Once trained and up to speed on [Critical reminders](#critical-reminders-for-all-linguists) (below), log in to the [Survey Tool](https://st.unicode.org/cldr-apps/) to begin your work.

### Survey Tool Changes

### Viewing / Adding Hidden Characters
When you are adding a value, you can see any "Hidden" characters, and insert additional ones.

> These include characters that are completely invisible, plus variants of spaces.

Reviewer 3: ```suggestion
These include characters that are completely invisible, as well as variants of spaces.
```

For example, in the image below, someone is in the middle of adding a new item. 

> There is a new "eye" icon in the bottom left that you can toggle to see hidden characters, 

Reviewer 3: ```suggestion
There is a new "eye" icon in the bottom left that the user can toggle to see hidden characters, 
```

and they have turned it on.

> That opens a new (uneditable) box below the text entry, where you can see the NBSP variant of the space character.

Reviewer 3: ```suggestion
That opens a new (uneditable) box below the text entry, where they can see the NBSP (non-breaking space) variant of the space character.
```

They realize that they need to insert a hidden character, 
so they pull down the insert-character menu from the new "insert-character" icon in the top left.
That lets them insert a character at the current insertion point in the text.

> Hovering over the items in that menu gives details about their usage, as you see in the image.

Reviewer 3: ```suggestion
Hovering over the items in that menu shows details about their usage, as you see in the image.
```

<img width="668" height="349" alt="Show Hidden When Adding" src="https://github.com/user-attachments/assets/52253a53-7105-431b-b3db-4a6e2cec02c3" />

> NOTE: For Alphabetic information (such as exemplar characters), an older mechanism is still in place.  

Reviewer 3: ```suggestion
NOTE: For alphabetic information (such as exemplar characters), an older mechanism is still in place.  
```
